contingent_plan_executor/local_run_utils.py

```diff
@@ -19,7 +19,6 @@ def run_rasa_model_server(output_files_path):
         requests.post("http://localhost:5005/model/parse", json={"text": ""})
     except requests.exceptions.ConnectionError:
         subprocess.Popen(f"rasa run --enable-api -m {output_files_path}/nlu_model.tar.gz", shell=True)
-        sleep(3.0)
```

Comment:
Hrmz...what was this here for before?

Reply:
On my home PC it always took a few seconds for the rasa model to set up so I just added that so it wouldn't try to run things a million more times than needed when I was testing things and further slow things down. Just a silly hack I forgot to remove (promise that's the only one 😋)